lib/Wikia/tests/Util/SamplerProxyTest.php

```diff
@@ -0,0 +1,297 @@
+<?php
+
+use Wikia\Util\SamplerProxy;
+
+class SamplerProxyTest extends \WikiaBaseTest {
+
+	/** @var \PHPUnit_Framework_MockObject_MockObject */
+	private $originalMock;
+
+	/** @var \PHPUnit_Framework_MockObject_MockObject */
+	private $alternateMock;
+
+	private $enableShadowing = true;
+	private $methodSamplingRate = 50;
+
+	private $originalMethodToSample = 'methodToSample';
+	private $originalMethodNotToSample = 'methodNotToSample';
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$originalMethodNotToSample" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1068)